app/src/main/java/net/gsantner/opoc/util/StringUtils.java

```diff
@@ -345,4 +347,35 @@ public static void selectLines(final EditText edit, final List<Integer> position
             edit.setSelection(selStart, selEnd);
         }
     }
+
+    // Search for matching pairs of backticks
+    // interpolate contents of backtick pair as SimpleDateFormat
+    public static String interpolateEscapedDateTime(final String snip) {
+        final StringBuilder interpolated = new StringBuilder();
+        final StringBuilder temp = new StringBuilder();
+        boolean isEscaped = false;
```

Comment:
that is part of the snippets PR, isn't it?

Reply:
I built this on top of the snippets PR because I knew there would be some conflicts otherwise. This PR is WIP partially because I am waiting to close the snippets work before cleaning up and working on this PR.